manager_janitor.go

```diff
@@ -335,6 +335,105 @@ func (mgr *Manager) pindexesRestart(
 	return errs
 }
 
+func (mgr *Manager) activatePIndex(pi *pindexRestartReq) (*PIndex, error) {
```

Comment:
Forgot the flow here. 
ActivatePIndex name hints that the pindex would already be closed, why are stopping it again?